Python/bin/sexVals.py

```diff
@@ -324,9 +324,9 @@ def isoToSexDate(isodate, prec):
           sexdate
        Errors:  Only if prec is wierd
    """
-
-   isoformat = "%Y-%m-%dT%H:%M:%S.%fZ"
-   dt = datetime.strptime(isodate, isoformat)
+    
+   # Convert input time to datetime object
+   dt = datetime.fromisoformat(isodate)
```

Comment:
Before Python 3.11, `fromisoformat` doesn't support the `Z` suffix to indicate that the datetime is UTC. Inputs with the `Z` suffix will throw `ValueError: Invalid isoformat string`. The `Z` suffix seems ubiquitous in usage though. Is support for older versions of Python desired? If so, we will need a more complex implementation here.

Reply:
Thanks, @stevenstetzler! And I've since learned that `fromisoformat` is not available for v3.6 and earlier. I'm not sure how far back in version we should try to keep functional, but requiring v3.11 seems too strict.

Reply:
@stevenstetzler I've pushed a new version that does not rely in `fromisoformat` and I think should work for all python3 versions. Please take a look. It tests fine here, but I'm on v3.12.